core/src/test/scala/unit/kafka/log/LogLoaderTest.scala

```diff
@@ -1800,5 +1822,6 @@ class LogLoaderTest {
       isRemoteLogEnabled = isRemoteLogEnabled
     ).load()
     assertEquals(expectedLogStartOffset, offsets.logStartOffset)
+    log.close()
```

Comment:
what happens if the assertion in above line fails? we will have a leak in that case.

Reply:
good point! I think I addressed it.